### DISCUSSION TITLE
add UEFA.EURO

### DISCUSSION THREAD

**Discussion by Federico83-hassio** [vasqued2/ha-teamtracker#157] [Q&A]
Hi everyone, I'm trying to integrate the European competition UEFA.EURO but I can't integrate my team in my case Italy, can you help me integrate it?

**Comment by vasqued2** [comment]
Use custom configuration w/ `sport_path` set to `soccer` and `league_path` set to `UEFA.EURO`

Team should be `ITA` for Italy.

**Reply by Federico83-hassio** [reply]
not working 
![image](https://github.com/vasqued2/ha-teamtracker/assets/58251354/75d0ffbe-b271-440a-bde7-0b2d0ea5be32)

**Reply by vasqued2** [reply]
That's because you used `ITALY` instead of `ITA`.

**Reply by Federico83-hassio** [reply]
I followed what you wrote here https://github.com/vasqued2/ha-teamtracker/issues/160#issuecomment-2170468633
and it worked, while if I do it from integration it doesn't work by putting ITA
![image](https://github.com/vasqued2/ha-teamtracker/assets/58251354/2491a14d-b559-449d-b787-5b5c924246d2)
![image](https://github.com/vasqued2/ha-teamtracker/assets/58251354/827a7f38-6e8c-4595-bbcf-e888a753630f)

**Reply by vasqued2** [reply]
ITA works for me from the GUI.  If you post a screenshot of all of the values that isn't cut off I might be able to figure it out.  Otherwise you will have to post the logs to troubleshoot.

**Comment by Federico83-hassio** [comment]
![Screenshot_20240616-175641](https://github.com/vasqued2/ha-teamtracker/assets/58251354/a889fecd-e73a-4eca-b63e-0b11450537ec)
![Screenshot_20240616-175708](https://github.com/vasqued2/ha-teamtracker/assets/58251354/7e8230fa-6df3-4c15-96c5-7e62a29b4c78)
![Screenshot_20240616-175721](https://github.com/vasqued2/ha-teamtracker/assets/58251354/ca256989-e37b-4e31-8728-af438dacc094)
Where am I wrong?

**Reply by Federico83-hassio** [reply]
Ok I found the problem is the conference number, left blank now it works 
![Screenshot_20240616-180306](https://github.com/vasqued2/ha-teamtracker/assets/58251354/05cd3094-f81c-42b1-89f0-b575f952b42f)
